samples/test-samples.cmake

```diff
@@ -50,6 +51,15 @@ else ()
   set(SAMPLES_LIST host_verify)
 endif ()
 
+if (BUILD_ENCLAVES)
+  set(SAMPLES_LIST helloworld file-encryptor log_callback switchless)
+  # Debug malloc will set allocated memory to a fixed pattern.
+  # Hence do not enable pluggable_allocator test under USE_DEBUG_MALLOC.
+  if (COMPILER_SUPPORTS_SNMALLOC AND NOT USE_DEBUG_MALLOC)
+    list(APPEND SAMPLES_LIST pluggable_allocator)
+  endif ()
+endif ()
+
 if ($ENV{OE_SIMULATION})
```

Comment:
Can you verify that the sample gets packed and installed? Thanks

Reply:
Also add this sample to https://github.com/openenclave/openenclave/blob/ac4169d199906905414980a7fad60a56f16fe58a/docs/GettingStartedDocs/install_oe_sdk-Simulation.md#considerations